driver/compile_common.mli

```diff
@@ -48,12 +48,13 @@ val with_info :
 val parse_intf : info -> Parsetree.signature
 (** [parse_intf info] parses an interface (usually an [.mli] file). *)
 
-val typecheck_intf : info -> Parsetree.signature -> Typedtree.signature
+val typecheck_intf :
+  info -> Parsetree.signature -> Misc.alerts * Typedtree.signature
 (** [typecheck_intf info parsetree] typechecks an interface and returns
     the typedtree of the associated signature.
 *)
 
-val emit_signature : info -> Parsetree.signature -> Typedtree.signature -> unit
+val emit_signature : info -> Misc.alerts -> Typedtree.signature -> unit
 (** [emit_signature info parsetree typedtree] emits the [.cmi] file
     containing the given signature.
```

Comment:
@nojb this PR introduced a change to the type of those two functions, and the documentation comments are now out of date. Would you be willing to refresh the documentation comments? (I think that a direct push to trunk would be perfectly fine for this.) I could do it myself, but I'm not completely sure how do describe the alerts returned and the alerts passed. Just let me know if you don't have time, I will go at it.

Reply:
Yes, will do shortly. Thanks for the ping!

Reply:
See https://github.com/ocaml/ocaml/commit/43f56ad5543f69ef796d9960096457ff27c8b9da